static/js/components/content/link_chips.tsx

```diff
@@ -38,28 +38,34 @@ export interface LinkChip {
 }
 
 interface LinkChipsProps {
+  //the variant of the link chip to display: elevated is a raised grey chip and flat is a flat blue chip
+  variant?: "elevated" | "flat";
   //the title of the component, displayed as a header above the chips
   title?: string;
+  //the section gives location of the chip component in order to give context for the GA event
+  section: string;
   //the link
   linkChips: LinkChip[];
 }
 
 export const LinkChips = ({
+  variant = "elevated",
   title,
+  section,
   linkChips,
 }: LinkChipsProps): ReactElement => {
   return (
-    <section id="chip-section" className="chip-section">
+    <section className="chip-section">
       <div className="container">
         {title && <h3>{title}</h3>}
         <ul className="chip-container">
           {linkChips.map((linkChip) => (
-            <li key={linkChip.id} className="chip-item">
+            <li key={linkChip.id} className={`chip-item ${variant}`}>
               <a
                 href={linkChip.url}
                 onClick={(): void => {
                   triggerGAEvent(GA_EVENT_HOMEPAGE_CLICK, {
-                    [GA_PARAM_ID]: `topic ${linkChip.id}`,
+                    [GA_PARAM_ID]: `${section} ${linkChip.id}`,
                     [GA_PARAM_URL]: linkChip.url,
                   });
                   window.location.href = linkChip.url;
```

Comment:
It looks like command or control clicking on these links doesn't open the link in a new tab. Can you look into fixing that? We have some GA logging here, but im not sure that's strictly necessary - GA might automatically log all link clicks regardless.